### DISCUSSION TITLE
[question] how to connect to milvus remotely

### DISCUSSION THREAD

**Discussion by SummerXXXX** [milvus-io/milvus#6603] [Q&A and General discussion]
Hi, I'm new to milvus and docker, I'm trying to install milvus in a centos server and run the hello_milvus.py  from another win10 computer.
I have followed the [guide](https://milvus.io/docs/v2.0.0/install_standalone-helm.md) and installed successfully.

![Screenshot 2021-07-17 180534](https://user-images.githubusercontent.com/10771754/126033514-7069b86a-eaa6-4d5c-a5ea-e52d6f7521c3.png)

after follow the guide to port-forward milvus service and then I try to run the hello_milvus.py from my computer, I have change the connection host:
![Screenshot 2021-07-17 175808](https://user-images.githubusercontent.com/10771754/126033339-706ed312-b030-4fdd-801a-11d434e3a16b.png)

but I still got an connection error:
![Screenshot 2021-07-17 175828](https://user-images.githubusercontent.com/10771754/126033356-f216b141-7941-4a5c-84e6-c8be0e06a971.png)

Becase I'm new to milvus and docker , so I'm not sure is this because I need some additional configuration? like docker port expose?
Thanks for your help.

**Comment by SummerXXXX** [comment]
I start minikube with command  "minikube start --listen-address=0.0.0.0"
https://minikube.sigs.k8s.io/docs/faq/#how-can-i-access-a-minikube-cluster-from-a-remote-network

**Reply by xiaofan-luan** [reply]
what you can do is run telnet ip 19530 and see if network is reachable. 
If not then you probably has to expose your pod to public. 
Kubectl expose is probably the command you are looking for
